index.html

```diff
@@ -36,193 +48,290 @@
+            ></a>
+          </li>
+          <li>
+            <a href="https://www.linkedin.com/in/chiedozie013"
+              ><i class="fab fa-linkedin"></i
+            ></a>
+          </li>
+        </ul>
       </section>
-      <section class="works-section">
-        <h2>My Recent Works</h2>
-        <img src="./images/Img Placeholder.png" alt="workImg" />
-        <div class="workDetails">
-          <h3>Multi-Post Stories</h3>
-          <p>
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required. has been the industry's standard dummy text ever
-            since the 1500s, when an unknown printer took a standard dummy text.
-          </p>
-          <ul>
-            <li>CSS</li>
-            <li>HTML</li>
-            <li>Bootstrap</li>
-            <li>Ruby</li>
-          </ul>
-          <button class="btn" href="#">See project</button>
+      <section id="portfolio">
+        <div class="works-section">
+          <h2>My Recent Works</h2>
+          <div class="works-vector"></div>
         </div>
-      </section>
-      <section>
-        <div class="workCard">
-          <h2>Professional Art Printing Data</h2>
-          <p>
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required. Has been the industry's standard.
-          </p>
-          <ul>
-            <li>HTML</li>
-            <li>Bootstrap</li>
-            <li>Ruby</li>
-          </ul>
-          <button class="btn" href="#">See project</button>
+        <div class="works">
+          <img src="./images/Img Placeholder.png" alt="workImg" />
+          <div class="workDetails">
+            <h3>Multi-Post Stories</h3>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. This has been the industry's standard dummy
+              text ever since the 1500s, when an unknown printer took a standard
+              dummy text.
+            </p>
+            <ul class="work-lang">
+              <li>CSS</li>
+              <li>HTML</li>
+              <li>Bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <button class="btn" href="#">See project</button>
+          </div>
         </div>
-        <div class="workCard">
-          <h2>Professional Art Printing Data</h2>
-          <p>
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required. Has been the industry's standard.
-          </p>
-          <ul>
-            <li>HTML</li>
-            <li>Bootstrap</li>
-            <li>Ruby</li>
-          </ul>
-          <button class="btn" href="#">See project</button>
+      </section>
+      <section class="works-main">
+        <div class="workCard1">
+          <div class="position">
+            <div class="hide-card">
+              <h2 class="card-title">Professional Art Printing Data</h2>
```

Comment:
- [ ] [Optional] I notice that in you desktop version there is 4 cards on each row but if you see the design then the 6 cards is divided into 2 rows. It is important to stick to the design as in the future you will make the exact design need by the users. Also, See the design below for reference.
- [ ] [Optional] If you see the design then only first card has black border and there is a gap between the cards. Please make sure to stick to the design as much as possible.
### Figma design
![figma](https://github.com/chiedozie013/portfolio-microverse/assets/81727988/cb2c027e-f9e9-49b5-9e55-9adee20ec557)

### Your design
![his](https://github.com/chiedozie013/portfolio-microverse/assets/81727988/8de586ad-7a62-4d1d-8ad6-ed45f36906ec)